src/app/dynamic-forms/service/print/print.service.ts

```diff
@@ -384,21 +392,6 @@ export class PrintService {
     } else if (value && currentField && currentField.cinchyColumn.dataFormatType === 'LinkUrl') {
       return `<a href="${value}" target="_blank">Open</a>`;
     }
-    else if(this.isAnchor(value)){
```

Comment:
I think we need this, this was code @cinchy-yogeshjagtap added to address a bug with rendering anchor elements... could it be that branch isn't up to date?

Reply:
This is an issue with the branch interactions. I've corrected this.